src/extra/python/isca/templates/compile.sh

```diff
@@ -20,6 +19,7 @@ netcdf_flags=`nc-config --fflags --flibs`
 # 2. Load the necessary tools into the environment
 module purge
 source {{ env_source }}
+source $python_package_lib_path
```

Comment:
What is contained within this source that is not possible to put into the environment file `src/extra/env/cheyenne`?

Reply:
The issue is this line change in `compile.sh`.  Could you remove it, and do the $python_package_lib_path thing in the NCAR specific env file?  It should have the same effect as this file is sourced in the line above